R/theme-current.R

```diff
@@ -91,9 +91,12 @@ get_theme <- function() {
 theme_get <- get_theme
 
 #' @rdname get_theme
-#' @param new new theme (a list of theme elements)
+#' @param new new theme (a list of theme elements). Sets theme to the default (theme_grey) if `new` isn't supplied.
```

Comment:
This should link to the default theme:

```suggestion
#' @param new new theme (a list of theme elements). Sets theme to the default ([theme_grey()]) if `new` isn't supplied.
```